lib/client/player_test.go

```diff
@@ -42,7 +42,7 @@ func TestEmptyPlay(t *testing.T) {
 	case <-p.stopC:
 	}
 
-	require.True(t, p.Stopped())
+	require.True(t, p.Stopped(), "p.Stopped() returned false when it should have returned true")
```

Comment:
This text seems unnecessary. The assertion is already going to give you an error message that says "expected true, got false" and a line number.

Reply:
Ok, changing it to precisely @codingllama 's suggestion here: https://github.com/gravitational/teleport/pull/11559#discussion_r838783706

Reply:
Sorry to give you the run around on this one, I didn't see the conflicting comment. I disagree with @codingllama, but he's a better developer than I am so let's take his suggestion 🙂

Reply:
The fun of multiple reviewers. :)

Feel free to pushback on nits if you disagree or think it's not worth the effort, I don't hold approvals for nits. I do like the descriptions, but I agree it's rather minor.